### PR TITLE
RS101 - Added codeowners

### PR DESCRIPTION
**Pull request by Adnanhoq**

### CODE REVIEW

File: .github/workflows/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@AZZohaib
```
Comment (TomBillingtonUK) [inline .github/workflows/CODEOWNERS:1]: Add the mentors as well please :)

Comment (Adnanhoq) [reply .github/workflows/CODEOWNERS:1]: Okay

**Review by Leebee1392 — DISMISSED** [review]

File: .github/workflows/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
```
Comment (sdagn1) [inline .github/workflows/CODEOWNERS:1]: You're missing yourself from this document.

**Review by sdagn1 — APPROVED** [review]

File: .github/workflows/CODEOWNERS (hunk 2 above)
Comment (Adnanhoq) [reply .github/workflows/CODEOWNERS:1]: Added

**Review by Leebee1392 — APPROVED** [review]

**Pull request merged**